### DISCUSSION TITLE
404 when using a path in the URI for docker hosting

### DISCUSSION THREAD

**Discussion by TheRealBenForce** [FreshRSS/FreshRSS#5968] [Q&A]
**Describe the bug**
When hosting with a path in the base_url, navigating to the url will produce a 404. Navigating to the url _without_ the path will redirect you to the path and produce a 302 followed by a 404.

**To Reproduce**
1. Starting with a base_url in the config.php file with no path, add a path to the end: `'base_url' => 'http://10.10.10.10:8085/rss',`
2. Stop, pull and restart docker containers.
3. Navigate to the base url with or without the new path the new path.

**Expected behavior**
FreshRss should be hosted from the path specified in the base url. I also wouldn't expect a redirect.

**Screenshots**
![image](https://github.com/FreshRSS/FreshRSS/assets/8939575/7b855003-1522-48a3-8429-979854c7c13f)

**Environment information (please complete the following information):**
- Device: PC
- OS: Windows 11
- Browser: Chrome 120.0.6099.72 
- FreshRSS version: 1.22.1 (freshrss/freshrss:latest)
- Database version: postgres:16
- PHP version: 8.2.7
- Installation type: Docker

**Additional Comments**
Some container logs:

***Before when hosted without a path and working as expected:***
10.10.10.2 - - [22/Dec/2023:17:22:41 +0000] "GET /favicon.ico HTTP/1.1" 200 18102 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"

***After adding a path and navigating to the base***
http://10.88.10.10:8085
I'm redirected here:
http://10.88.10.10:8085/rss/i/?rid=6585c62891aeb

***And these logs are produced:***
10.10.10.2 - - [22/Dec/2023:17:23:52 +0000] "GET / HTTP/1.1" 302 - "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
10.10.10.2 - - [22/Dec/2023:17:23:52 +0000] "GET /rss/i/?rid=6585c62891aeb HTTP/1.1" 404 196 "-" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"

**Comment by Alkarex** [comment]
Hello,
This is not how it works.
Please check https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#run-in-production

**Reply by TheRealBenForce** [reply]
Thanks @Alkarex , so long story short I think I was quite wrong in many ways. I thought I might take on the task of organizing all my containers under a single subdomain, but reviewing your link, seeing all the considerations for headers, and scaling that out to all the other containers I think I'll probably keep my current setup of hosting freshrss on its own subdomain.
